src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -1194,8 +1237,28 @@ function LineplotWithControls({
         max: truncationConfigDependentAxisMax,
       },
     },
+    independentAxisLogScale: vizConfig.independentAxisLogScale,
+    dependentAxisLogScale: vizConfig.dependentAxisLogScale,
   };
 
+  const [
+    dismissedIndependentAllNegativeWarning,
+    setDismissedIndependentAllNegativeWarning,
+  ] = useState<boolean>(false);
+  const independentAllNegative =
+    vizConfig.independentAxisLogScale &&
+    xMinMaxDataRange?.max != null &&
+    xMinMaxDataRange.max < 0;
+
+  const [
+    dismissedDependentAllNegativeWarning,
+    setDismissedDependentAllNegativeWarning,
+  ] = useState<boolean>(false);
+  const dependentAllNegative =
+    vizConfig.dependentAxisLogScale &&
+    yMinMaxDataRange?.max != null &&
+    yMinMaxDataRange.max < 0;
```

Comment:
less than or equal to 0 here? If the max is 0 we're still in a bad place with the log scale

Reply:
Yes, good catch!  I won't rename the variable though.